lib/matplotlib/widgets.py

```diff
@@ -2908,10 +2908,10 @@ class RectangleSelector(_SelectorWidget):
     ...     print(erelease.xdata, erelease.ydata)
     >>> props = dict(facecolor='blue', alpha=0.5)
     >>> rect = mwidgets.RectangleSelector(ax, onselect, interactive=True,
-                                          props=props)
+    ...                                   props=props)
     >>> fig.show()
-
-    >>> selector.add_state('square')
+    >>>
```

Comment:
```suggestion
```
?

Reply:
I suspected the extra line was there to emphasize it, but I'm not sure.

Reply:
It starts a new box, so probably, yes. I'm more wondering about the rationale to put an empty command-line/return-press instead. Better to drop it completely then?